client/components/social-buttons/google.js

```diff
@@ -45,6 +47,10 @@ class GoogleLoginButton extends Component {
 			return this.initialized;
 		}
 
+		this.setState( { error: '' } );
```

Comment:
Can it be initialized more than once? If not then we can remove this state change.